tests/results/TextPositionTest.py

```diff
@@ -30,3 +40,58 @@ def test_properties(self):
         uut = TextPosition(8, 39)
         self.assertEqual(uut.line, 8)
         self.assertEqual(uut.column, 39)
+
+
+class TextPositionLineColTest(unittest.TestCase):
+
+    def test_from_position(self):
```

Comment:
it's the same function that's being tested, and it does give the exact line which failed? yeah there two many tests maybe i can seperate a few not sure though